File: jetpack_java/src/main/java/com/kunminx/jetpack_java/sample_04_databinding/ui/state/LocationViewModel.java


package com.kunminx.jetpack_java.sample_04_databinding.ui.state;

import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

import com.kunminx.jetpack_java.common_data.bean.LocationBean;

import java.util.List;

/**
 * Create by KunMinX at 2020/5/30
 */
public class LocationViewModel extends ViewModel {

    public final MutableLiveData<List<LocationBean>> list = new MutableLiveData<>();


}

File: jetpack_kotlin/src/main/java/com/flywith24/jetpack_kotlin/sample_04_databinding/ui/state/LocationViewModel.kt
package com.flywith24.jetpack_kotlin.sample_04_databinding.ui.state

import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel
import com.flywith24.jetpack_kotlin.common_data.bean.LocationBean

/**
 * @author Flywith24
 * @date   2020/5/31
 * time   23:13
 * description
 */
class LocationViewModel : ViewModel() {
    val list = MutableLiveData<List<LocationBean>>()
}